portfolio/src/main/java/com/google/sps/servlets/CommentServlet.java

```diff
@@ -31,59 +28,60 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-
 /** Servlet that returns comments */
 @WebServlet("/comment")
 public class CommentServlet extends HttpServlet {
 
   /** Comment data structure */
-  protected class Comment{
+  protected class Comment {
     private final String text;
     private final String user;
     private final long timestamp;
 
-    public Comment(String user, String text, long timestamp){
+    public Comment(String user, String text, long timestamp) {
       this.text = text;
       this.user = user;
       this.timestamp = timestamp;
     }
-
   }
 
-  /** Adds all the comments in the DataStore to the response argument, in descending chronological order */
+  /**
+   * Adds all the comments in the DataStore to the response argument, in descending chronological
+   * order
+   */
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     Query query = new Query("comment").addSort("timestamp", SortDirection.ASCENDING);
-    
+
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     PreparedQuery results = datastore.prepare(query);
 
     ArrayList<Comment> comments = new ArrayList<Comment>();
 
-    for(Entity comment : results.asIterable()){
+    for (Entity comment : results.asIterable()) {
       String user = (String) comment.getProperty("user");
       String text = (String) comment.getProperty("text");
       long timestamp = (long) comment.getProperty("timestamp");
 
       comments.add(new Comment(user, text, timestamp));
     }
-    
-    Gson gson  = new Gson();
+
+    Gson gson = new Gson();
     response.setContentType("application/json;");
     response.getWriter().println(gson.toJson(comments));
   }
 
   /** Adds a comment to the DataStore with data parsed from the body of the request */
   @Override
-  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException{
-    
-    // TODO: Check if user is logged in before doing everything else 
-    
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+    // TODO: Check if user is logged in before doing everything else
```

Comment:
TODOs are usually formatted with an issue number or username, so something like "TODO(tabaresj): Check if user is logged in"

Reply:
All right, thank you!